packages/zowe-explorer/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - Power users and developers can now build links to efficiently open mainframe resources in Zowe Explorer. Use the **Copy External Link** option in the context menu to get the URL for a data set or USS resource, or create a link in the format `vscode://Zowe.vscode-extension-for-zowe?<ZoweResourceUri>`. For more information on building resource URIs, see the [FileSystemProvider wiki article](https://github.com/zowe/zowe-explorer-vscode/wiki/FileSystemProvider#file-paths-vs-uris). [#3271](https://github.com/zowe/zowe-explorer-vscode/pull/3271)
 - Implemented more user-friendly error messages for API or network errors within Zowe Explorer. [#3243](https://github.com/zowe/zowe-explorer-vscode/pull/3243)
 - Use the "Troubleshoot" option for certain errors to obtain additional context, tips, and resources for how to resolve the errors. [#3243](https://github.com/zowe/zowe-explorer-vscode/pull/3243)
+- Added the ability to `Search Filtered Data Sets` on Data Set session nodes, as well as the ability to `Search PDS Members` on Partitioned Data Sets. This enables users to easily search for data inside of their Data Sets. [#3306](https://github.com/zowe/zowe-explorer-vscode/pull/3306)
 
```

Comment:
Can we make this more action oriented?: "Easily search for data in data sets and partitioned data sets with the new Search functionality."